webpack_loader/config.py

```diff
@@ -25,6 +30,11 @@
     for name, cfg in user_config.items()
 )
 
+if user_config['DEFAULT']['ASSETS_LOADER']['func'] == DEFAULT_CONFIG['DEFAULT']['ASSETS_LOADER']['func']:
```

Comment:
Oh here's a potential gotcha - by building the dictionary like
 
    user_config = dict(
        (name, dict(DEFAULT_CONFIG['DEFAULT'], **cfg))
        for name, cfg in user_config.items()
    )
`user_config['DEFAULT']['ASSETS_LOADER']` will be a _reference_ to `settings.WEBPACK_LOADER['DEFAULT']['ASSETS_LOADER'] or `DEFAULT_CONFIG['DEFAULT']['ASSETS_LOADER']`, not a copy, and anything from the default config `ASSETS_LOADER` dict won't make it into the one in `user_config` if the `ASSETS_LOADER` key is in `settings.WEBPACK_LOADER['DEFAULT']` (potentially with a customised func but not args).
I think this will still work in our user case, but whether it will work in all use cases is not obvious.